subliminal/providers/itasa.py

```diff
@@ -0,0 +1,390 @@
+# -*- coding: utf-8 -*-
+import copy
+
+import io
+
+import logging
+
+from babelfish import Language
+
+from requests import Session
+
+from zipfile import ZipFile, is_zipfile
+
+from . import ParserBeautifulSoup, Provider, get_version
+
+from .. import __version__
+
+from .. cache import SHOW_EXPIRATION_TIME, region
+
+from .. exceptions import AuthenticationError, ConfigurationError, TooManyRequests
+
+from .. subtitle import (Subtitle, fix_line_ending, guess_matches, guess_properties, sanitize_string,
+                         sanitized_string_equal)
+
+from .. video import Episode
+
+logger = logging.getLogger(__name__)
```

Comment:
* Group from/imports by "kind" (native modules imports, dependencies imports, relative imports) and sort them alphabetically with one new line between each group.
* Sort imports alphabetically (first constants, then classes, then functions).

See other modules for example. This is not critical but just to keep the style consistent.